src/patternfly/components/FormControl/form-control.scss

```diff
@@ -72,6 +77,13 @@ $pf-c-form-control--success--Coordinates: "M504 256c0 136.967-111.033 248-248 24
   --pf-c-form-control--success--check--Background: var(--pf-c-form-control--success--BackgroundUrl) var(--pf-c-form-control--success--BackgroundPosition) / var(--pf-c-form-control--success--BackgroundSize) no-repeat;
   --pf-c-form-control--success--Background: var(--pf-c-form-control--BackgroundColor) var(--pf-c-form-control--success--check--Background);
 
+  // Input m-search
+  --pf-c-form-control--m-search--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-form-control--m-search--BackgroundPosition: var(--pf-c-form-control--PaddingLeft);
+  --pf-c-form-control--m-search--BackgroundSize: var(--pf-c-form-control--FontSize) var(--pf-c-form-control--FontSize);
```

Comment:
Should the background-size use font-size or line-height?

Reply:
I copied these settings from the previous examples